srcpkgs/qsv/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'qsv'
 pkgname=qsv
-version=2.2.1
+version=3.2.0
 revision=1
 archs="x86_64* i686* aarch64* arm*" # ring
 build_style=cargo
 configure_args="--features=feature_capable"
 make_check_args="-- --skip=test_comments --skip=test_sample --skip=test_sort
  --skip=test_describegpt"
-hostmakedepends="pkg-config"
+hostmakedepends="cmake pkg-config"
```

Comment:
`gzp` dependency grabs and builds `libz-ng-sys` by default, but we could probably get by with just the rust backend if we disabled default features in Cargo.toml `gzp`

Reply:
nevermind, the above is already true and yet `libz-ng-sys` is still used, even when I don't see why 😱

Reply:
oh good, I couldn't manage to figure it out